generators/entity-client/index.js

```diff
@@ -59,6 +64,31 @@ module.exports = class extends BaseBlueprintGenerator {
     return this._default();
   }
 
+  _loadEntityConfig() {
+    this.paginationPagination = this.entity.pagination === PAGINATION;
+    this.paginationInfiniteScroll = this.entity.pagination === INFINITE_SCROLL;
+    this.paginationNo = this.entity.pagination === NO_PAGINATION;
+
+    this.searchEngineFalse = this.entity.searchEngine === false;
+  }
+
+  _loading() {
+    return {
+      loadSharedConfig() {
+        this.loadAppConfig();
+        this.loadClientConfig();
+        this.loadServerConfig();
+        this.loadTranslationConfig();
```

Comment:
There is one detail about entity-client:
- at gateways, entities from microservices, should load the configuration of the microservice not the gateway one.

So IMO should be removed.

Reply:
Removed because it was already in generator-entity.
I think that there's something to do on that part: we can't execute entity-client in a standalone mode (without executing entity). It looks against good code principles but can also be annoying for blueprints.
I can raise a ticket with my suggestion following what I encounter during the constant refactoring